src/style.css

```diff
@@ -1,5 +1,102 @@
 @import url('https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600&family=Lato:wght@100;300;400;700;900&family=Poppins:wght@100;200;300;400;500;600;700;800;900&family=Raleway:wght@100;200;300;400;500;600;700;800;900&family=Roboto:wght@100;300;400;500;700;900&family=Source+Sans+Pro:wght@200;300;400;600;700&display=swap');
 
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
   font-family: 'Inter', sans-serif;
+  line-height: 1.8;
+  font-size: 16px;
+  color: rgba(255, 255, 255, 1);
+  background-color: rgba(10, 10, 10, 1);
+}
+
+button {
+  padding: 4px;
+  cursor: pointer;
+  max-width: 70px;
+  border-radius: 6px;
+}
+
+section {
+  max-width: 550px;
+  margin: 0 auto;
+  margin-top: 60px;
+  padding: 5px;
+  background-color: rgba(255, 255, 255, 0.1);
+  backdrop-filter: blur(10px);
+  border-radius: 15px;
+}
+
+.content {
+  padding: 10px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.main-title {
+  padding: 0 10px;
+}
+
+/* left side styling */
+.left-side {
+  width: 50%;
+}
+
+.left-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 10px;
+}
+
+.refresh-btn {
+  background-color: rgba(239, 151, 44, 1);
+  border: 1px solid rgba(239, 151, 44, 1);
+  transition: all 0.3s ease;
+  cursor: pointer;
+}
+
+.refresh-btn:hover {
+  background-color: transparent;
+  color: rgba(239, 151, 44, 1);
+}
+
+table {
+  width: 100%;
+  border-collapse: collapse;
+  border-spacing: 0;
+  border: none;
+}
```

Comment:
- [ ] Kindly notice that the table is showing the scores out of the container, and that making it not like the wireframe please fix that.
- [ ] Also there is a horizontal scroll please fix that too.

Your Design | Wireframe
------------- | ----------------
![Capture](https://user-images.githubusercontent.com/33752735/202796588-f23a9441-d7a7-487c-81ce-d3a74311b3fb.PNG) | ![image](https://user-images.githubusercontent.com/33752735/202796679-75a50237-cc55-4d4f-86a5-3b56749c243d.png)